src/psyclone/tests/dynamo0p3_transformations_test.py

```diff
@@ -3898,11 +3898,10 @@ def test_rc_discontinuous_depth(tmpdir, f90, f90flags):
     invoke = psy.invokes.invoke_list[0]
     schedule = invoke.schedule
     rc_trans = Dynamo0p3RedundantComputationTrans()
-    loop = schedule.children[3]
+    loop = schedule.children[0]
```

Comment:
No initial halo exchanges for the continuous fields as the annexed dofs are now up-to-date by default.

Reply:
Yes that's right.